docs/api-reference/geo-layers/tile-layer.md

```diff
@@ -266,8 +267,22 @@ Receives arguments:
 
 Receives arguments:
 
-- `tile` (Object) - the tile that has been cleared from cache.
+- `tile` (Object) - the [tile](#tile) that has been cleared from cache.
+
+## Tile
+
+Class to hold the reading of a single tile
+
+Properties:
+
+- `x` (Number) - x index of the tile
+- `y` (Number) - y index of the tile
+- `z` (Number) - z index of the tile
+- `bbox` (Object) - bounding box of the tile. When used with a geospatial view, `bbox` is in the shape of `{west: <longitude>, north: <latitude>, east: <longitude>, south: <latitude>}`. When used with a non-geospatial view, `bbox` is in the shape of `{left, top, right, bottom}`.
+- `data` (Array) - tiles content as returned by `getTileData`. 
 
 ## Source
 
 [modules/geo-layers/src/tile-layer](https://github.com/visgl/deck.gl/tree/master/modules/geo-layers/src/tile-layer)
+
+
```

Comment:
I'm surprised the extra ending whitespace passed the linter.

Reply:
Does the linter work in markdown?

Reply:
I've definitely gotten linter errors in Markdown in loaders.gl; I assumed it would be the same linter in deck.gl